sdk/typescript/src/signers/signer-with-provider.ts

```diff
@@ -98,7 +98,6 @@ export abstract class SignerWithProvider implements Signer {
     );
 
     return {
-      // TODO: Should this include the intent, or just be the raw bytes that are signed?
```

Comment:
Just going to say for now that the message handed back doesn't include intent, and that the `verifyMessage` is expected to attach the intent. It seems less error prone and prevents `verifyMessage` from being used to verify things like transaction data.